files/english/ORGANIZATION-DAY.md

```diff
@@ -1 +1,22 @@
-# Organization on the day
\ No newline at end of file
+# Organization on the day
+
+>Here are some instructions and tips to be followed by the PUG-PE organizing members and volunteers on the event's day.
+
+## Instructions
+
+### Some things to be aware be of before the event's day
+
+It is important to the organizing member and volunteers to check if there are some items that might be necessary, just like: table, power plugs, video projector. It is wanted that, in case those items are not available, the members should look for alternate solutions so the meetup can happen in the best possible way for everyone.
```

Comment:
Acredito que o uso do "e.g." em língua escrita é mais comum e funciona melhor do que o "just like". O que tu acha?

```suggestion
It is important to the organizing member and volunteers to check if there are some items that might be necessary (e.g., table, power plugs, video projector). It is wanted that, in case those items are not available, the members should look for alternate solutions so the meetup can happen in the best possible way for everyone.
```